src/components/productDetails/card.jsx

```diff
@@ -71,10 +71,10 @@ const Card = ({
       </div>
       <div className="card-content">
         <div className="media-content">
-          <p className="title is-3">{product.category}</p>
+          <p style={{"border-bottom": ".5px solid black"}} className="">{product.category}</p>
```

Comment:
maybe add to a css file if you think it could change later

Reply:
Good idea. Wanted to brute force it to work and then make changes later.